app/src/androidTest/java/com/sdp/movemeet/view/map/GPSRecordingActivityTest.java

```diff
@@ -62,9 +85,10 @@
     @Rule
     public GrantPermissionRule CoarseLocationPermissionRule = GrantPermissionRule.grant(Manifest.permission.ACCESS_COARSE_LOCATION);
 
+    Activity activity;
 
-    @Rule
-    public ActivityScenarioRule<GPSRecordingActivity> testRule = new ActivityScenarioRule<>(GPSRecordingActivity.class);
+    /*@Rule
+    public ActivityScenarioRule<GPSRecordingActivity> testRule = new ActivityScenarioRule<>(GPSRecordingActivity.class);*/
```

Comment:
Why this rule is commented ?

Reply:
Ideally, I would like to be able to launch the Scenario for this test class without having to set it up in a @Before function, but it requires passing an Activity as an extra and I'm unsure of how to go about that exactly